resources/dicts/events/death/beach/apprentice.json

```diff
@@ -1,3 +1,225 @@
+        "history_text": {
+                        "reg_death": "m_c succumbed to dehydration and overheating."
+                        },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null
+     },
+     {
+        "camp": "any",
+        "tags": [
+            "Newleaf", 
+            "Greenleaf", 
+            "Leaf-fall",
+            "other_cat",
+            "other_cat_mentor"
+        ],
+        "death_text": "r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, round black beetles crawling in, out, and across {PRONOUN/m_c/poss} once lively pelt.",
+        "history_text": {
+                        "reg_death": "m_c died due to dehydration and overheating."
+                        },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null  
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_adult"
+        ],
+        "death_text": "r_c realizes {PRONOUN/r_c/subject} {VERB/r_c/haven't/hasn't} seen m_c around camp for a few days. Checking {PRONOUN/m_c/subject} favorite spots yields nothing, so r_c decides to go out and search for {PRONOUN/m_c/object} in the Clans' territory. However, r_c could never have expected to find m_c's shriveled-up body, clearly dead for some time now.",
+        "history_text": {
+                     "reg_death": "m_c died due to dehydration and heat exhaustion."
+                 },
+        "cat_trait": null,
+         "cat_skill": null,
+         "other_cat_trait": null,        
+         "other_cat_skill": null 
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_parent"
+        ],
+        "death_text": "r_c is frantic with worry; m_c, {PRONOUN/r_c/poss} child, hadn't been seen around camp, no matter who {PRONOUN/r_c/subject} asked and where {PRONOUN/r_c/subject} searched. It's not until r_c stumbles across m_c's body, withered under the sun and being feasted on by black and orange butterflies, with a despairing sound, that {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} {PRONOUN/r_c/poss} greatest fears have come true.",
```

Comment:
Similar issue here.  I suggest revising "It's not until r_c stumbles across m_c's body, withered under the sun and being feasted on by black and orange butterflies, with a despairing sound, that {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} {PRONOUN/r_c/poss} greatest fears have come true."